libcodechecker/libhandlers/parse.py

```diff
@@ -278,21 +326,23 @@ def main(args):
             files = [os.path.join(input_path, file_name) for file_name
                      in file_names]
 
+        file_change = False
         for file_path in files:
-            report_stats = parse(file_path,
-                                 context,
-                                 metadata_dict,
-                                 suppress_handler,
-                                 skip_handler,
-                                 'print_steps' in args)
+            report_stats, f_change = parse(file_path,
+                                           context,
+                                           metadata_dict,
+                                           suppress_handler,
+                                           skip_handler,
+                                           'print_steps' in args)
+            if not file_change:
```

Comment:
I might be a bit confused here. Isn't `f_change` a list? I just tested and empty list evaluates to false, but do we want to rely on this?

Reply:
Yes `file_change` is a list and using and an empty list will evaluate to `False` this is a know and used python behavior.